mvpa2/algorithms/hyperalignment.py

```diff
@@ -182,6 +186,13 @@ class Hyperalignment(ClassWithCollections):
             updated common space, and is subsequently called again after each
             2nd-level iteration.""")
 
+    joblib_backend = Parameter(None, constraints=EnsureChoice('multiprocessing',
+                                                    'threading') | EnsureNone(),
+            doc="""Backend to use for joblib when using nproc>1.
```

Comment:
and here commend `nproc != 1` ... although not sure if we should check silly folks specifying 0 ;)